TESTS/mbed_platform/minimal-printf/compliance/main.cpp

```diff
@@ -42,6 +42,114 @@
 
 using namespace utest::v1;
 
+#define MAX_STRING_SIZE 100
+// Buffer used by test cases to store a test string
+static char expected_string[MAX_STRING_SIZE];
+
+#define BASE_10 10
```

Comment:
I do not think these defines are necessary. Passing `10` and `16` as values for the function parameter `base` is very clear already.